packages/typescript/chat-core/test/ai/use-ai-chat.test.ts

```
@@ -0,0 +1,406 @@
+import { describe, it, expect, beforeEach, afterEach, mock, spyOn } from 'bun:test';
```

ESLint

[error] 'mock' is defined but never used.

(@typescript-eslint/no-unused-vars)